frontend/src/services/AdvisorService.ts

```diff
@@ -1,5 +1,18 @@
 import { getAuthToken } from "../utils/auth-helpers";
 
+export interface IAbrStudent {
+  fullName: string;
```

Comment:
I added a IAbrUser that's very similar to this to the types folder. Can we merge the two?

Reply:
Seems like they're for two different use cases, so I renamed the existing one to `IAbrAdvisor`